src/main/java/dev/thedocruby/resounding/raycast/Fix.java

```diff
@@ -25,10 +25,9 @@
 import static dev.thedocruby.resounding.config.PrecomputedConfig.pC;
 
 @Environment(EnvType.CLIENT)
-public class RaycastFix {
-
-    private RaycastFix() {}
+public class Fix {
```

Comment:
I'm okay with you removing the "resounding", but try make this class name a bit more descriptive. I get that it is in the "raycast" class, but still... Maybe "Patch" would more accurate and easier to understand at a glance.

Reply:
Renamed to Patch.